deepmd/dpmodel/fitting/make_base_fitting.py

```
@@ -67,6 +67,30 @@ def compute_output_stats(self, merged: Any) -> NoReturn:
             """Update the output bias for fitting net."""
             raise NotImplementedError
 
+        def reinit_exclude(self, exclude_types: list[int] = []) -> None:
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)